client/my-sites/customer-home/index.js

```diff
@@ -9,9 +9,19 @@ import page from 'page';
 import { navigation, siteSelection, sites } from 'my-sites/controller';
 import home, { maybeRedirect } from './controller';
 import { makeLayout, render as clientRender } from 'controller';
+import { trackNavigationStart } from 'lib/performance-tracking';
 
 export default function () {
-	page( '/home', siteSelection, sites, makeLayout, clientRender );
+	page( '/home', trackNavigationStart( 'home' ), siteSelection, sites, makeLayout, clientRender );
```

Comment:
What kind of time interval do we want to track? More precisely, what is the moment when the time measurement starts?

When the `trackNavigationStart` is placed in this route handler, the sequence of events is:
1. User triggers navigation in browser, e.g., by clicking a sidebar link
2. The page.js router starts handling the `/home` route.
3. The first handler to run is `/home/*` from the `section-loader`, which asynchronously loads the `home` webpack chunk from network. That takes some time.
4. The downloaded chunk's JS code runs and registers the `page( '/home' )` handler that I'm commenting on.
5. The page.js router continues to call the registered handlers for `/home`, finally getting to the newly registered one.
6. Only now `trackNavigationStart` gets called, quite some time after the user initiated the navigation.
7. The React UI for `/home` is rendered. That's usually part of the `home` chunk, unless it involves async-loaded React components.
8. As part of the React rendering, `<PerformanceTrackerStop />` is mounted.

Between the start and stop events, there is only JS execution and no network is involved (depending on how and when exactly `<PerformanceTrackerStop />` is rendered. The measured times will be in milliseconds.

Reply:
Oh, I totally missed the `section-loader` :( Thanks for the tip, I'll investigate how to instrument it then